lib/registry.sh

```diff
@@ -0,0 +1,39 @@
+function activate-docker-machine-env {
+  eval "$(docker-machine env deis-registry)"
+
+  save-vars DOCKER_TLS_VERIFY DOCKER_HOST DOCKER_CERT_PATH DOCKER_MACHINE_NAME
+}
+
+function create-dev-registry {
+  (
+    cd ${DEIS_ROOT}
+    rerun_log warn "Creating local Docker registry..."
+    make dev-registry > /dev/null
+  )
```

Comment:
Yeah - they create a subshell so that I return to the rigger working directory after the make call.